.travis.yml

```diff
@@ -2,32 +2,32 @@ language: python
 cache: pip
 
 python:
-  - "3.5"
-  - "2.7"
+  - "3.6"
+  - "3.7"
 
 env:
-  - DJANGO="django>=1.11,<1.12"
-  - DJANGO="django>=2.1,<2.2"
-  - DJANGO="django>=2.2,<2.3"
-
-matrix:
-  exclude:
-   - python: "2.7"
-     env: DJANGO="django>=2.1,<2.2"
-   - python: "2.7"
-     env: DJANGO="django>=2.2,<2.3"
+  - DJANGO="django>=2.2,<3.0"
+  - DJANGO="django>=3.0,<3.1"
 
 branches:
   only:
     - master
 
+before_install:
+  - pip install -U pip
+  - pip install -U pip wheel setuptools
+
 install:
   - pip install $DJANGO
+  - pip install -e git+git://github.com/openwisp/django-netjsongraph#egg=django-netjsongraph
```

Comment:
I suggest locking it to some specific commit otherwise, the build will probably fail with changes in `django-netjsongraph`.

Reply:
don't worry, I will handle this once I release django-netjsongraph